config/routes.rb

```diff
@@ -1,10 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root "home#index"
```

Comment:
I'm planning to use a reverse proxy like Nginx, which handles HTTPS termination, so I don't think there's a need to connect to the Rails app using HTTP/2.

Reply:
HTTP/2 has a number of benefits, including the ability to handle far more WebSocket connections per HTTP network connection. HTTP/2 between the browser and the server must use TLS, but between servers it could be plain text HTTP/2. I'm not sure if nginx supports that. 

Whether that is useful or not will depend on your scale requirements. Maybe you can comment on the number of active users you are hoping to support and the kind of deployment environment (number of servers, server hardware, etc) and I can give you more feedback.